roles/kubespray-defaults/defaults/main/main.yml

```diff
@@ -62,11 +62,6 @@ kubeadm_join_phases_skip: >-
 # Set to true to remove the role binding to anonymous users created by kubeadm
 remove_anonymous_access: false
 
-# Supported asymmetric encryption algorithm types for the cluster's keys and certificates.
-# can be one of RSA-2048(default), RSA-3072, RSA-4096, ECDSA-P256
-# ref: https://kubernetes.io/docs/reference/config-api/kubeadm-config.v1beta4/#kubeadm-k8s-io-v1beta4-ClusterConfiguration
-kube_asymmetric_encryption_algorithm: "RSA-2048"
-
```

Comment:
You should probably mention in the commit message why we can do that (== since the switch makes it used only in one role), otherwise it can appear a bit random to the uninitiated ^.